FirebaseInAppMessagingDisplay.podspec

```diff
@@ -23,15 +23,8 @@ Firebase In-App Messaging SDK.
   s.static_framework = true
```

Comment:
Why isn't this file deleted?

Reply:
We're keeping it in as an empty framework. This is to handle the case where a developer pulls in the new consolidated SDK, but leaves in the display SDK. They would get duplicate symbols since all the display classes are now in the consolidated one.

We're going to leave the display SDK in as an empty framework, there is a warning that tells developers to delete it.

Reply:
Add s.deprecated_in_favor_of = 'FirebaseInAppMessaging'. See https://guides.cocoapods.org/syntax/podspec.html